dependencies.yaml

```diff
@@ -18,6 +19,14 @@ files:
     pyproject_dir: .
     extras:
       table: build-system
+    includes:
+      - rapids_build_skbuild
+  py_rapids_build:
+    output: pyproject
+    pyproject_dir: .
+    extras:
+      table: tool.rapids-build-backend
+      key: requires
     includes:
       - build
       - build_tools
```

Comment:
If you want this to look a bit more like other RAPIDS libraries, then by convention these 2 lists would be spelled like this:

```yaml
  py_build_legateboost:
    output: pyproject
    pyproject_dir: .
    extras:
      table: build-system
    includes:
      - rapids_build_skbuild
  py_rapids_build_legateboost:
    output: pyproject
    pyproject_dir: .
    extras:
      table: tool.rapids-build-backend
      key: requires
    includes:
      - build
      - build_tools
```

There are some docs on those conventions at https://github.com/rapidsai/build-planning/blob/d9e3c606d95c835ee384ac6480a4af0ac6cb024a/docs/docs/packaging.md#L85

Not critical, but following RAPIDS conventions does make it easier for more other RAPIDS folks to come in and help out on these repos.